### DISCUSSION TITLE
Is it possible to simplify the editor (like by removing the Link tool)?

### DISCUSSION THREAD

**Discussion by amaury-hanser** [codex-team/editor.js#2645] [Q&A]
Hello,  

I have a project where I want to give the user the ability to format some text to a certain extent.

1. I'd like if possible to remove the Link from the tools so the user can't insert any link in it's content. Is it possible to remove it from the inline toolbar?
2. In the MVP of the project, I'd like the user to only be able to add Paragraph block. Is it possible to remove the whole Block tool so any new block would be a paragraph?

I'm considering EditorJS because we already use it in our monorepo, but I'm wondering if it's truly the right choice for me.
Any help would be appreciated, thanks!

**Comment by neSpecc** [comment]
Yes. 

1. Specify the [`inlineToolbar`](https://editorjs.io/configuration/#inline-toolbar-order) option
2. Omit the `tools` property or pass an empty object

**Comment by Matt-Eva** [comment]
The problem I was running into was disabling the `Link` tool without disabling the ability to convert text to headers and back. If I only passed `["bold", "italic"]` to my inlineToolbar property, it would get rid of the ability to convert text to headers and back. The only formatting I need is the ability to bold, italicize, and convert text to headers and back.

My solution to this was a bad CSS hack - I selected the button with the class `ce-inline-tool--link` (which I found by combing through my browser devtools) and setting its `display` to `none`. This has effect that I need, even though it's not using editorJS itself to remove the link feature.
